# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1455122774 18000
#      Wed Feb 10 11:46:14 2016 -0500
# Node ID ffc9806bb30048658ced9820f3d76c3753c6710a
# Parent  5d23ea5c733ad0b41b02e11c5c1d99b1f14a766b
mk-octave-config-h.sh: Correct typdef for octave_idx_type.

diff --git a/build-aux/mk-octave-config-h.sh b/build-aux/mk-octave-config-h.sh
--- a/build-aux/mk-octave-config-h.sh
+++ b/build-aux/mk-octave-config-h.sh
@@ -87,17 +87,17 @@ octave_idx_type="`sed -n 's/#define OCTA
 
 if test -z "$octave_idx_type"; then
   echo "mk-octave-config-h.sh: failed to find OCTAVE_IDX_TYPE in $config_h_file" 1>&2
   exit 1
 fi
 
 cat << EOF
 
-typedef octave_idx_type $octave_idx_type;
+typedef $octave_idx_type octave_idx_type;
 
 EOF
 
 if grep "#define OCTAVE_ENABLE_FLOAT_TRUNCATE 1" $config_h_file > /dev/null;
 then
   echo "#define OCTAVE_FLOAT_TRUNCATE volatile"
 else
   echo "#define OCTAVE_FLOAT_TRUNCATE"
